src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/BuildPropertyPage.VisualBasic.xaml.cs.xlf

```diff
@@ -305,42 +305,42 @@
       <trans-unit id="EnumValue|OptionCompare.Binary|DisplayName">
         <source>Binary</source>
         <target state="translated">Binary</target>
-        <note />
+        <note>{Locked}</note>

```

Comment:
Did you edit these manually? I suspect that changes to the XAML will cause these files to be regenerated, losing your changes here. If I'm not mistaken, doing this for these files will first require adding comment support to xliff-tasks.

Something like:

```xml
    <EnumValue Name="Text" DisplayName="Text">
      <!-- XliffComment: {Locked} -->
    </EnumValue>
```

Reply:
Smit and I were talking about this with Yanni; we also suspect that's the case, but we wanted to try and see if this works, as we think what's overwritten is the target (translated values) and not the notes (comments). But we're not 100% sure :)

Reply:
I suspect it just re-writes the whole file, but I'm also not sure. It will be easy to verify.

The concern here is that GitHub collapses the XLF files and so future work could easily revert these improvements and slip through review without being noticed.

Reply:
This is true, these changes are ephemeral